src/charm.py

```diff
@@ -122,7 +123,8 @@ class _ReplicaRelationNotReady(Exception):
     state = StoredState()
 
     def __init__(self, *args, **kwargs):
-        """Initialize the instance."""
+        # Ignore arguments for Charmbase in docstring.
+        """Initialize the instance."""  # noqa: E731, DCO020
```

Comment:
What are these rules?

Reply:
These rules were to ignore the *args and **kwargs, but i've added proper docstring to it addressing what *args and **kwargs are :) thank you for the comment!